src/ui/windows/TogglDesktop/TogglDesktop/ui/ViewModels/TimelineViewModel.cs

```diff
@@ -365,20 +367,42 @@ public class TimeEntryBlock : ReactiveObject
         [Reactive]
         public bool IsEditViewOpened { get; set; }
 
-        public TimeEntryBlock(string timeEntryId)
+        private double _hourHeight;

```

Comment:
Zooming support?

Reply:
Zooming support is not necessary here, as all the TimeEntryBlok's are re-created when zooming changes and there will be a new TimeEntryBlock with new hourHeight.